### PR TITLE
SN Update annotated_filenames

### PR DESCRIPTION
**Pull request by sarahgonicholson**
In `commands/create_annotated_filenames.py`:
- For annotated filenames, update `aliquot_id` to be `[aliquot_id]MC` if multiple tissue samples from the same tissue aliquot but multiple cores (e.g. ST001-1A-001A1 and ST001-1A-001B2) and `MAMC` if from multiple tissue samples from different tissue aliquots
- Remove the variant type from the end of annotated filenames for vcfs

In `commands/write_submission_spreadsheets.py`
- Add notes for properties that allow commas for large numbers and allow abbreviations (e.g. 100M or 3.1 Gb)

In `schemas/file.json`
- Add a `version` property for front-end

### CODE REVIEW

File: src/encoded/commands/create_annotated_filenames.py [hunk 1]
```diff
@@ -654,11 +650,29 @@ def get_aliquot_id_from_samples(tissue_samples: List[Dict[str], Any]) -> Filenam
...
         get_aliquot_id_from_tissue_sample(sample) for sample in tissue_samples
     ]
+    if len(aliquot_ids) > 1:
+        aliquot_ids = get_multiple_aliquot_id_from_samples(aliquot_ids)
     return get_filename_part_for_values(
         aliquot_ids, "tissue aliquot ID", source_name="sample"
     )
 
 
+def get_multiple_aliquot_id_from_samples(ids: List[str]):
```
Comment (aschroed) [inline src/encoded/commands/create_annotated_filenames.py:660]: As I am not clear on how the decision on this ended up and the desired result I leave it up to you to make sure that this is doing as expected.

**Review by aschroed — APPROVED** [review]
Looks fine to me.

File: src/encoded/commands/create_annotated_filenames.py [hunk 2]
```diff
@@ -654,11 +650,29 @@ def get_aliquot_id_from_samples(tissue_samples: List[Dict[str], Any]) -> Filenam
...
+    """Get filename part for files merged from multiple tissue sample aliquots."""
+    aliquot_ids = []
+    core_ids = []
+    for id in ids:
+        aliquot_id = id[:-2]
+        core_id = id[-2:]
```
Comment (willronchetti) [inline src/encoded/commands/create_annotated_filenames.py:666]: You have an implicit invariant here that must be true in order for calls like this to succeed - so long as there is external validation elsewhere that ensures this, it's probably fine, but may merit a comment to illustrate what you are extracting.

File: src/encoded/schemas/file.json [hunk 3]
```diff
@@ -207,6 +210,10 @@
                 "title": "Notes to tsv file",
                 "type": "string"
             }
+        },
+        "version": {
+            "type": "string",
+            "permission": "restricted_fields"
```
Comment (willronchetti) [inline src/encoded/schemas/file.json:216]: Intended UX for this? May merit some further discussion on how we want to manage "versions" of files

**Review by willronchetti — APPROVED** [review]
Needs version bump/CHANGELOG

File: src/encoded/schemas/file.json (hunk 3 above)
Comment (sarahgonicholson) [reply src/encoded/schemas/file.json:216]: @willronchetti This is intended for the COLO829 Challenge data/Truth set, where presumably there might be multiple challenges and we want to keep track of which one the files came from. Open to suggestions on other ways to handle this though!

**Pull request merged**